common-lib/src/main/java/com/google/gct/idea/feedback/FeedbackUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.google.gct.idea.feedback;
+
+import com.intellij.ExtensionPoints;
+import com.intellij.ide.plugins.PluginManager;
+import com.intellij.openapi.extensions.Extensions;
+import com.intellij.openapi.extensions.PluginId;
+
+/**
+ * Handy plugin related methods for Google Feedback integration.
+ */
+public final class FeedbackUtil {
+
+  private FeedbackUtil() {}
+
+  public static void enableGoogleFeedbackErrorReporting(String pluginId) {
```

Comment:
I don't know if this will work, but I would try to write a test that calls this and see what happens. You might further be able to check that the error reporter is registered. May not work, but worth a shot.